config.yml.example.swift

```diff
@@ -1,12 +1,12 @@
 development: &defaults
   maintenance: false
-  # # uncomment this section
-  # swift: &swift_defaults
-  #   host: "https://swift.example.com"
-  # # Redis is needed for the swift backend
-  # redis:
-  #   host: localhost
-  #   port: 6379
+  # uncomment this section
```

Comment:
The section is already uncommented? I think now that the examples are seperate, this comment is obsolete.